### ISSUE TITLE
nav bar bug

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wcag-redesign#2]
navigation scroll vs wrap.

with large text or small screen, the template gets a scrollbar (mine has custom big teal-green scrollbars):
<img width="500" alt="nav scrollbar fyi-mine-is-designed-bigger" src="https://user-images.githubusercontent.com/6991814/185007168-cfb7b3c9-67ff-402d-b706-05eae414bb55.png">

WCAG's goes to next line under visual bar, and sometimes breaks across a line “Next SC: <br>Handle”:
<img width="500" alt="nav wrap" src="https://user-images.githubusercontent.com/6991814/185007277-0d8c9ddd-2b20-48d7-b840-d3ff1cc2e570.png">

**Comment by michael-n-cooper** [comment]
I'll need the exact HTML, CSS, and Javascript code that should be output to address this.

**Comment by SteveALee** [comment]
see #27

**Comment by SteveALee** [comment]
I saw the scrollbar whenever hover ove the pager so suspect a width bug somewhere

**Comment by shawna-slh** [comment]
deleting this issue so it's all in one place: https://github.com/w3c/wcag-redesign/pull/27
